src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/TransferOnlineCaseCallbackHandler.java

```diff
@@ -133,6 +133,8 @@ private CallbackResponse saveTransferOnlineCase(CallbackParams callbackParams) {
                         featureToggleService.isLocationWhiteListedForCaseProgression(newCourtLocation.getEpimmsId())
                             ? YesOrNo.YES : YesOrNo.NO);
                 }
+            } else if (isLipCaseWithProgressionEnabledAndCourtWhiteListed(caseData)) {
```

Comment:
Again, same here

Reply:
Also, double check the location, it gets saved on line121 in the builder, but the old casedata won't have been updated yet.

Reply:
Done